ms-ledger/routes/common_test.go

```diff
@@ -14,6 +14,11 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const(
+	// LedgerFileName is the filename for ledger.json file
```

Comment:
maybe self explanatory?

Reply:
Correct, don't need to comment code that is self descriptive.

Reply:
yep, I have removed that comment